Storage/test.cc

```diff
@@ -48,7 +54,7 @@ namespace transactional_storage_test {
 CURRENT_STRUCT(Element) {
   CURRENT_FIELD(x, int32_t);
   CURRENT_CONSTRUCTOR(Element)(int32_t x = 0) : x(x) {}
-  // Dummy key function to support key-only deleter.
+  // To use `Element` in storage, it should either define a `.key` field, or `.key()` method.
```

Comment:
This is actually a bug/feature: `Element` is only used in `Vector`, which doesn't need the `key/key()`. We'll take care of it soon-ish.